lasagne/updates.py

```diff
@@ -54,9 +54,6 @@ def get_or_compute_grads(loss_or_grads, params):
 def sgd(loss_or_grads, params, learning_rate):
     """Stochastic Gradient Descent (SGD) updates.
 
-    Generates update expressions of the form:
-    * ``param := param - learning_rate * gradient``
-
```

Comment:
Why did you remove these explanations? I thought they were useful to compare momentum and nesterov momentum (and to be explicit about the formulation we use, as there is another momentum formulation that scales the learning rate by `1 - momentum`). I'm not sure if that was the best place to put them, but I don't think they should just be dropped. Are they confusing/unneeded?